hack/release-notes.tmpl

```diff
@@ -0,0 +1,18 @@
+# Changelog since {{ .PreviousRevision }}
+
+{{with .NotesWithActionRequired -}}
+## Urgent Upgrade Notes
+
+### (No, really, you MUST read this before you upgrade)
+
+{{range .}} {{println "-" .}} {{end}}
+{{end}}
+
+{{- with .NotesByKind -}}
```

Comment:
This requires us to set kind on every PR properly. Would you please adjust the PR template to do that? To have a required preset with /kind feature, /kind bug (as done in KubeVirt) and ask the person opening the PR to select one.

Reply:
Where do we have that doc ?